lib/text.js

```diff
@@ -9,13 +9,16 @@ const wrapBold = ({bold}, el) => bold ? <b>{el}</b> : el;
 
 const wrapItalic = ({italic}, el) => italic ? <i>{el}</i> : el;
 
+const wrapStrikethrough = ({strikethrough}, el) => strikethrough ? <s>{el}</s> : el;
```

Comment:
I get that this is a clean one-liner, but I think as an outsider it makes this really confusing to have the arrow function AND the ternary in one line.